survey-runner/terraform.tfvars.example

```diff
@@ -1,27 +1,27 @@
-aws_access_key="XXXXXXXXXXXX"
-aws_secret_key="XXXXXXXXXXXX"
-aws_key_pair="XXXXX"
-elastic_beanstalk_aws_key_pair="XXXXX"
-ons_access_ips=[XXXXXXX, XXXXXXX]
-certificate_arn="arn:aws:iam::XXXXXXX:server-certificate/NAME"
+aws_access_key="XXXXXXXXXXXX" # Update with access key
+aws_secret_key="XXXXXXXXXXXX" # Update with secret key
+aws_key_pair="XXXXX" # Set to the name of pem file (Not including .pem)
+elastic_beanstalk_aws_key_pair="XXXXX" # Set to the name generated ssh key in aws
+ons_access_ips=[XXXXXXX, XXXXXXX] # Set to the IP addresses you will be accessing accessing survey runner from
+certificate_arn="arn:aws:iam::XXXXXXX:server-certificate/NAME" # Set to arn of the tls certificate used by survey runner
 application_secret_key="you'll never guess it"
 vpc_ip_block="10.30.20.0/24"
-rabbitmq_ip_prime="XX.XX.XX.15"
-rabbitmq_ip_failover="XX.XX.XX.16"
+rabbitmq_ip_prime="10.30.20.15"
+rabbitmq_ip_failover="10.30.20.16"
 google_analytics_code="UA-56892037-7"
 rabbitmq_ips = {
-  "0" = "XX.XX.XX.15"
-  "1" = "XX.XX.XX.16"
+ "0" = "10.30.20.15"
+ "1" = "10.30.20.16"
 }
 logserver_cidr="x.x.x.21/32"
 audit_cidr="x.x.x.242/32"
 sdx_cidr="x.x.x.0/32"
-application_cidr="10.30.2X.0/26"
-database_1_cidr="10.30.2X.64/26"
-database_2_cidr="10.30.2X.128/26"
+application_cidr="10.30.20.0/26"
+database_1_cidr="10.30.20.64/26"
+database_2_cidr="10.30.20.128/26"
 dev_mode="True"
 rsyslogd_server_ip="10.172.x.x"
-schema_bucket="eq-schema-files-preprod"
+schema_bucket="eq-schema-files-XXXXX" # Set to your schema bucket
```

Comment:
Not sure if it's being used for the author but can probably go if author in its current form is dead